test/replication-luatest/gh_11115_reconnect_timeout_test.lua

```diff
@@ -0,0 +1,81 @@
+local t = require('luatest')
```

Comment:
Do we use ticket number in the test name even for features? (I would consider this new option a feature.) I see this often in new patches, and perhaps I am not aware if we decided to change the naming policy.

Reply:
Nah, we don't, my bad